lib/reservation_manager.rb

```diff
@@ -0,0 +1,111 @@
+require 'date'
+require_relative 'reservation_dates'
+require_relative 'reservation'
+
+module HotelBookings
+  class Reservation_Manager
+    MAX_ROOMS = 20
+    attr_reader :customer_name, :checkin, :checkout
+   
+    def initialize(customer_name:)
+      @customer_name = customer_name
+
+      @current_reservations = {1 => [], 2 => [], 3 => [], 4 => [], 5 => [], 6 => [], 7 => [], 8 => [], 9 => [], 10 => [], 11 => [], 12 => [], 13 => [], 14 => [], 15 => [], 16 => [], 17 => [], 18 => [], 19 => [], 20 => []}
+    end 
+
+    def rooms 
+      return @current_reservations
+    end 
+
+    def reservation_list(date)
+      unavailable_rooms = []
+      key = 1
+      MAX_ROOMS.times do
```

Comment:
Is it possible to refactor this `MAX_ROOMS.times` to `@current_reservations.each do |room, reservations|`, so that `room` will always represent the value of the room? (aka below you would use `@current_reservations[room]`)